### PR TITLE
Updates suggested by Jake, plus minor fixes

### PR DESCRIPTION
**Pull request by samdutton**

### CODE REVIEW

File: service-worker/prefetch-video/index.html [hunk 1]
```diff
@@ -7,8 +7,9 @@
 
 <video controls  poster="static/poster.jpg"
   style="margin: 0 0 1em 0; max-width: 100%; width: 480px;">
-  <source src="static/gbike.webm" type="video/webm" />
-  <source src="static/gbike.mp4" type="video/mp4" />
+  <!-- Videos are stored remotely with CORS enabled -->
```
Comment (beaufortfrancois) [inline service-worker/prefetch-video/index.html]: We may want to specify that only `https://googlechrome.github.io` has been allowed in CORS Headers to not confuse developers who may try to it this URL and wondering why it doesn't work for them.

**Comment by beaufortfrancois** [comment]
LGTM but one nit.

**Comment by beaufortfrancois** [comment]
@samdutton Is this PR still valid?

**Comment by samdutton** [comment]
Hmm... Not sure what's happened here. 

AFAICT the only code that doesn't seem to be in the actual live demo at https://googlechrome.github.io/samples/service-worker/prefetch-video is the `console.log()` call — but Jeff may not have wanted to use that.

Looks like we should abandon this PR.

@jeffposnick Does that make sense?

**Comment by jeffposnick** [comment]
I'm fine with abandoning if that's your recommendation.

**Pull request closed without merging**